gcloud/bigquery/job.py

```diff
@@ -129,6 +129,15 @@ def __init__(self, name, client):
         self.name = name
 
     @property
+    def type(self):
```

Comment:
I wanted it to be attr-like;  I don't think there is a standard "class equivalent of a property" descriptor implementation.